persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml

```diff
@@ -230,9 +230,9 @@
                         </foreach>
                     </when>
                     <otherwise>
-                        AND (<include refid="caseStableIdentifier" />, genetic_profile.STABLE_ID) IN
+                        AND (genetic_profile.STABLE_ID, <include refid="caseStableIdentifier" />) IN
```

Comment:
this improved the performance by 10x

Reply:
How can this improve performance. BTW not questioning your work, just wondering why this causes a performance increase....

Reply:
Even I am not sure, but I think it might be related to index/key on the table and the way they have been joined.

Reply:
Actually I figured it by comparing to old query in mutationMapper.xml and see what exactly has changed, and found that these two params have been swapped.

Reply:
Great work! I would have never thought of this.